src/PooledArrays.jl

```diff
@@ -98,15 +99,23 @@ end
 _widen(::Type{UInt8}) = UInt16
 _widen(::Type{UInt16}) = UInt32
 _widen(::Type{UInt32}) = UInt64
-
+_widen(::Type{Int8}) = Int16
+_widen(::Type{Int16}) = Int32
+_widen(::Type{Int32}) = Int64
 # Constructor from array, invpool, and ref type
 
 """
-    PooledArray(array, [reftype])
+    PooledArray(array, [reftype]; signed=false, compress=false)
 
 Freshly allocate `PooledArray` using the given array as a source where each
 element will be referenced as an integer of the given type.
 If no `reftype` is specified one is chosen automatically based on the number of unique elements.
```

Comment:
This isn't correct now. See https://github.com/JuliaData/PooledArrays.jl/pull/57.